design-proposals/dra.md

```diff
@@ -142,24 +142,26 @@ type DeviceStatus struct {
 type DeviceStatusInfo struct {
 	// Name of the device as specified in spec.domain.devices.gpus.name or spec.domain.devices.hostDevices.name
 	Name string `json:"name"`
-	// DeviceResourceClaimStatus reflects the DRA related information for the device
-	// +optional
-	DeviceResourceClaimStatus *DeviceResourceClaimStatus `json:"deviceResourceClaimStatus,omitempty"`
```

Comment:
The idea for having this additional indentation was to keep the API flexible if devices need status information outside of DRA status.   Is this kind of flexibility out-of-scope for this proposal?